shared/boosting.lua

```diff
@@ -124,11 +125,11 @@ Config.Boosting.Locations = {
 
             peds = {
                 [1] = { coords = vector4(-815.58, -1120.26, 10.55, 307.23), health = 100, armor = 100,
-                    model = 'a_m_m_genfat_01', weapon = "weapon_bat" },
+                    model = 'a_m_m_genfat_01', weapon = Config.Boosting.PedSettings["C"].weapons[math.random(1, #Config.Boosting.PedSettings["C"].weapons)] },

```

Comment:
Also not sure if you knew this or not but, id the min is 1 you dont need to put 1 so `math.random(#Config.Boosting.PedSettings["C"].weapons)` would work

Reply:
i didn't know and tbh i just copied and pasted what's in the D class !